db/migrate/20220121115536_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] It will be nice to use `:author` as the foreign_key here instead of `:user`. The reason is given above.

Reply:
This issue is well resolved! ✔️